plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKudu113SmokeTests.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.kudu;
+
+public class TestKudu113SmokeTests
+{
+    private static final String KUDU_VERSION = "1.13.0";
+
+    public static class TestKuduSmokeTestWithDisabledInferSchema
```

Comment:
I'm surprised this kind of composition of tests actually works.

I wouldn't mind these being separate test classes but I'll defer to @Praveen2112 / @ebyhr.

Reply:
It looks neat for me. If there are no downsides we could keep it.

Reply:
I was surprised it worked too, but it is nice to encapsulate all the version-specific tests in one place.